back/app/serializers/web_api/v1/static_page_serializer.rb

```diff
@@ -1,10 +1,46 @@
 # frozen_string_literal: true
 
 class WebApi::V1::StaticPageSerializer < WebApi::V1::BaseSerializer
-  attributes :title_multiloc, :code, :slug, :created_at, :updated_at
+  attributes :title_multiloc,
```

Comment:
Should we conditionally exclude attributes that don't make sense for certain custom pages here? Such as that an FAQ cannot have projects_enabled, widgets_enabled etc?

Reply:
We agreed to make FAQ/About pages as flexible as custom ones https://citizenlabco.slack.com/archives/CQZGXTFGD/p1661182550902739
So, they can have projects and widgets.